server/routes/upload-document.route.js

```diff
@@ -219,6 +219,9 @@ module.exports = [
     path: `${Paths.UPLOAD_DOCUMENT}`,
     handler: handlers.post,
     config: {
```

Comment:
Add comment here and in the corresponding place in upload-photo to explain why this is required and that no text boxes should appear on this upload page

Reply:
Added